Document @datadog/apps-backend utilities for backend functions (#38457)
DataDog/documentation: content/en/actions/datadog_apps.md

The library is a fully typed TypeScript client that wraps integrations, including AWS, Azure, GCP, the Datadog API, GitHub, GitLab, Slack, Jira, PagerDuty, ServiceNow, OpenAI, Anthropic, and generic HTTP. Importing actions from `@datadog/action-catalog` gives you typed inputs and responses for each action.

> The [@datadog/apps-backend][23] package provides utilities to help with common tasks in your backend functions, such as retrieving the user who initiated the action.

Author: ```suggestion
You can view backend utilities through the [@datadog/apps-backend][23] package. Use utilities to help with common actions.
```

[20]: https://github.com/datadog-labs/agent-skills/tree/main/dd-apps/datadog-app
[21]: https://github.com/datadog-labs/agent-skills/blob/main/README.md
[22]: https://github.com/antfu/skills-cli

> [23]: 

Author: ```suggestion
[23]: https://www.npmjs.com/package/@datadog/apps-backend
```
